docs/0-Preparing/learning_about_industry.md

```diff
@@ -0,0 +1,31 @@
+## Learning about the industry
+
+An internship is a great way to learn real-world skills and earn some income to help pay tuition. They're also one of the primary ways to *get your foot in the door* of the tech industry. When you learn what the people in the industry want and care about, it becomes much easier to navigate it and speak the "insider" language. That is, you want to learn the *rules of the game*.
+
+In a lot of industries, working inside the industry is pretty much the only way to learn anything about the industry. That's why in other fields, there's a lot of entry-level positions with menial work that people take just to get their foot in the door.
+
+Fortunately for us, tech is unique that there's tons and tons of information on the Internet that people have put out freely. You'll find hiring managers talking openly about their feelings on recruiting in online discussions for example. Reading about tech isn't going to be *as* good as actually working in tech, but you can get pretty darn far.
+
+### Getting acquainted with the industry
+
+In the next section I'll provide some suggested readings. The point isn't to read all of it, and certainly not to cram this right before applying to jobs (though if you like reading, it can be addictive). The point is to keep an ear out for conversations that happen on the public sphere.
+
+A decent way to start is just to read for 30 minutes every other day. It can even be done when there's nothing better to do: e.g. when waiting for a class to start or when taking the bus. Even two hours of reading a week compounds to a lot over a year.
+
+> Rudi says: While it happened by accident, perhaps one of the most useful habits I got into was make reading my default procrastination mode. That is, whenever I get bored and take out my phone, I automatically open up the latest tech news/discussions.
+
+Note that as is typical of the Internet, you'll have to sift through a lot of wrong claims and garbage to get to the useful ideas. Keep that in mind, but it's not always a bad thing. In order to understand why something is "true", you also need to know what the alternatives were. Furthermore, knowing where people disagree can help you make balanced decisions and empathize.
+
```

Comment:
I'll add that. That reminds me, I'll add a note that the reading are heavily biased towards tech as in Sillicon Valley tech (which is fine since lots of other industries that make use of software just don't have as much of an online presence, so one needs a starting point somewhere)